Index: init.rb
===================================================================
--- init.rb	(revision 7)
+++ init.rb	(working copy)
@@ -1,11 +1,14 @@
 # Make sure that ActiveScaffold has already been included
 ActiveScaffold rescue throw "should have included ActiveScaffold plug in first.  Please make sure that this overwrite plugging comes alphabetically after the ActiveScaffold plug in"
 
+# set our directory
+$:.unshift(File.dirname(__FILE__))
+
 # Load our overrides
-load 'config/core.rb'
-load 'config/config_list.rb'
-load 'actions/config_list.rb'
-load 'helpers/view_helpers.rb'
+Kernel::load 'lib/config/core.rb'
+Kernel::load 'lib/config/config_list.rb'
+Kernel::load 'lib/actions/config_list.rb'
+Kernel::load 'lib/helpers/view_helpers.rb'
 
 ##
 ## Run the install script, too, just to make sure
Index: lib/actions/config_list.rb
===================================================================
--- lib/actions/config_list.rb	(revision 7)
+++ lib/actions/config_list.rb	(working copy)
@@ -73,11 +73,11 @@
     end
     
     def session_columns
-      show = @session[ self.class.to_s + '_col']
+      show = session[ self.class.to_s + '_col']
 
       if show.nil? || show.size == 0
         show = active_scaffold_config.config_list.default_columns.map{|c| c.to_s }
-        @session[self.class.to_s + '_col'] = show
+        session[self.class.to_s + '_col'] = show
       end
       
       show
@@ -98,7 +98,7 @@
       end
     
       if c.size > 0
-        @session[self.class.to_s + '_col'] = c       
+        session[self.class.to_s + '_col'] = c       
       end
      
     end
Index: lib/helpers/view_helpers.rb
===================================================================
--- lib/helpers/view_helpers.rb	(revision 7)
+++ lib/helpers/view_helpers.rb	(working copy)
@@ -2,9 +2,9 @@
   module Helpers
     module ViewHelpers
       # Add the cf_as plugin includes
-      def active_scaffold_includes_with_cf_as
-        css = stylesheet_link_tag(ActiveScaffold::Config::Core.asset_path('cf_as-stylesheet.css'))
-        ie_css = stylesheet_link_tag(ActiveScaffold::Config::Core.asset_path('cf_as-stylesheet-ie.css'))
+      def active_scaffold_includes_with_cf_as(frontend = :default)
+        css = stylesheet_link_tag(ActiveScaffold::Config::Core.asset_path('cf_as-stylesheet.css', frontend))
+        ie_css = stylesheet_link_tag(ActiveScaffold::Config::Core.asset_path("cf_as-stylesheet-ie.css", frontend))
         active_scaffold_includes_without_cf_as + "\n" + css + "\n<!--[if IE]>" + ie_css + "<![endif]-->\n"
       end
       alias_method_chain :active_scaffold_includes, :cf_as
Index: lib/config/core.rb
===================================================================
--- lib/config/core.rb	(revision 7)
+++ lib/config/core.rb	(working copy)
@@ -6,8 +6,10 @@
 
     # the active_scaffold_aal template path
     def template_search_path_with_cl_as
-      search_path = ["../../vendor/plugins/#{ActiveScaffold::Config::Core.cl_as_plugin_directory}/frontends/default/views"]
-      search_path += self.template_search_path_without_cl_as
+      frontends_path = "../../vendor/plugins/#{ActiveScaffold::Config::Core.cl_as_plugin_directory}/frontends"
+      search_path = template_search_path_without_cl_as
+      search_path << "#{frontends_path}/#{frontend}/views" if frontend.to_sym != :default
+      search_path << "#{frontends_path}/default/views"
       return search_path
     end
 
